src/Modules/UngradedModules/DirectSum.jl

```diff
@@ -70,7 +73,10 @@ Additionally, return
 - two vectors containing the canonical projections and injections, respectively, if `task = :both`,
 - none of the above maps if `task = :none`.
 """
-function direct_product(M::ModuleFP{T}...; task::Symbol = :prod) where T
+function direct_product(M::ModuleFP{T}, Ms::ModuleFP{T}...; task::Symbol = :prod) where T
+  return direct_product([M, Ms...]; task)
+end
+function direct_product(M::Vector{<:ModuleFP{T}}; task::Symbol = :prod) where T
   F, pro, mF = direct_product([ambient_free_module(x) for x = M]..., task = :both)
```

Comment:
```suggestion
  F, pro, mF = direct_product([ambient_free_module(x) for x = M], task = :both)
```
Better to not splat here, but use the new method taking a vector

Reply:
I wasn't sure if this is indeed now available in all possible cases that could occur here.